src/components/page-details.jsx

```diff
@@ -43,12 +43,14 @@ export default class PageDetails extends React.Component {
 
   componentWillUnmount () {
     window.removeEventListener('keydown', this);
+    document.title = 'Scanner';
   }
 
   /**
    * @param {PageDetailsProps} previousProps
    */
   componentDidUpdate (previousProps) {
+    document.title = this.state.page ? `Scanner | ${this.state.page.url}` : 'Scanner';
```

Comment:
Not a big deal, but it might be best to add a method that actually sets the page title, that way `componentWillUnmount` and `componentDidUpdate` can both call it, and you don’t have to worry about future changes to the title needing to be made in several places.

```js
setTitle (page = null) {
  document.title = this.state.page ? `Scanner | ${this.state.page.url}` : 'Scanner';
}

componentWillUnmount () {
  window.removeEventListener('keydown', this);
  setTitle();
}

componentDidUpdate () {
  setTitle();
  // rest of existing method code here
}
```

Reply:
Totally makes sense to make those improvements.  Okay, added those changes.  Thanks, Rob!

*side note: Noticed that my commits for this PR are racking up.  For future contributing, is there a way I should condense all my commits, is that something y'all do on your end, or is it better to leave them as is?

Reply:
> For future contributing, is there a way I should condense all my commits, is that something y'all do on your end, or is it better to leave them as is?

I usually collapse all the commits for a PR into one when merging (GitHub calls this “squash & merge”). That way, it’s easy for other people to pull your code and test it when you’re still working (since you aren’t messing with the commit history), but the history on the `master` and `release` branches stays nice and tidy.